src/modules/dashboard/components/ActionsPage/VoteResultsWidget/VoteResultsWidget.css

```diff
@@ -0,0 +1,94 @@
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 320px;
+}
+
+.voteInfoContainer {
+  display: flex;
+  width: 85%;
+}
```

Comment:
I saw the spec, and I understand why you aligned the icon higher than the progress bar:

![Screenshot from 2021-03-15 22-31-45](https://user-images.githubusercontent.com/1193222/111218762-d43e8380-85df-11eb-9532-0fd0737d6c12.png)

But I still think this will look better if the icon (and only the icon -- not the avatars) are aligned on the same line as the progress bar:

![Screenshot from 2021-03-15 22-39-14](https://user-images.githubusercontent.com/1193222/111218841-f33d1580-85df-11eb-9199-f2f75ab3fcf7.png)

I achieved the above by applying `align-self` to the icon:
```js
.voteInfoContainer i {
  ...
  align-self: baseline;
}
```